src/components/List.js

```diff
@@ -11,6 +11,7 @@ const List = ({ results, setSearchTerm, searchTerm, token }) => {
   function checkTime(time) {
     return Date.now() - time <= 86400000; //number of milliseconds equal to 24 hours
   }
+
```

Comment:
One nice, seemingly not important thing to do in PRs, is to minimize changes like this. Just makes for one less file to review.

Reply:
What would be the best way to check for differences like this?

Reply:
I use GitLens in my vscode editor, and before I commit it shows me the diff), and I can adjust before I commit, etc. Otherwise, if I forget, I can just view the diff in my PR in github, and change it back and push again.

https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens

Reply:
Looks like this:
![image](https://user-images.githubusercontent.com/10378331/92180464-b0b6a700-ee0c-11ea-888e-ad5483337a69.png)

It's also nice to have linting rules set up that automatically take care of things like this for you. prettier/eslint